src/app/vault/add-edit.component.html

```diff
@@ -164,7 +164,7 @@ <h2 class="modal-title" id="cipherAddEditTitle">{{title}}</h2>
                                     </a>
                                 </div>
                                 <div class="d-flex">
-                                    <select class="form-control" id="loginUriMatch{{i}}" name="Login.Uris[{{i}}].Match"
+                                    <select class="form-control overflow-hidden" id="loginUriMatch{{i}}" name="Login.Uris[{{i}}].Match"
```

Comment:
It might be better to add overflow: hidden to all selectors rather than ad-hoc to problematic ones.

Reply:
Maybe, if it happens to another or becomes a problem we can do that then I think.